scripts/protoc.py

```diff
@@ -1,11 +1,43 @@
+#!/usr/bin/env python
+
+import glob
+import os
+import shutil
+
 from grpc_tools import command
 
 
 # TODO: Add this command to setup.py once it is created
-def compile_proto_files() -> None: 
+def compile_proto_files() -> None:
     """Builds all proto files existing in Fed-BioMed"""
     command.build_package_protos('.')
 
 
 if __name__ == "__main__":
-    compile_proto_files()
\ No newline at end of file
+    root_dir = os.path.abspath(os.path.join(__file__,
+                                            '..',
+                                            '..',))
+    proto_dir = os.path.abspath(os.path.join(__file__,
+                                             '..',
+                                             '..',
+                                             "fedbiomed",
+                                             "transport",
+                                             "protocols"))
+    print(f"Refreshing: {proto_dir} files")
+    os.chdir(proto_dir)
+
+    # removing old proto files
+    for f in glob.glob("*.py"):
+        print(f"- cleaning: {f}")
+        os.remove(f)
+    for f in glob.glob("*.pyi"):
+        print(f"- cleaning: {f}")
+        os.remove(f)
+    shutil.rmtree('__pycache__', ignore_errors=True)
```

Comment:
Hi @j-l-s, 

It is nice to print outputs but it is not necessary to clean `__pycache__` (they are not committed) and old `py` files. Modules are overwritten by `compile_proto_files`.  So I would keep only `Refreshing: ..` part to give feedback to user`.

Reply:
hi @srcansiz @j-l-s can I reviewed the part about conda envs. When you complete the part about protoc, you can merge.

Reply:
Hi @mvesin, 
Thank you. We've checked the issue that @j-l-s  is having on his machine, and the problem is not really about the `grpcio` or `grpc-tools` version. It works only when the version is not specified, and does not work if any version is specified.  So, I don't think that it is necessary to keep `...-arm.yaml` file for conda. However, it is nice to have upgraded version of grpc.